src/frontend/context/FagsakContext.tsx

```diff
@@ -33,7 +33,13 @@ const [FagsakProvider, useFagsakRessurser] = createUseContext(() => {
     const { axiosRequest } = useApp();
 
     React.useEffect(() => {
-        if (fagsakRessurser.fagsak.status === RessursStatus.SUKSESS) {
+        if (
```

Comment:
I recommend splitting this into two useEffects. One for fetching the fagsak, and another listening on changes to fagsak.

Once the fagsak is loaded and either the person is not fetched or the fagsak is connected to another person we fetch the correct person. 

We can also replace fagsakRessurser with two separate useStates like we do with logg 👍

Reply:
Updated👍